src/js/Content/Features/Store/App/FShowCoupon.js

```diff
@@ -14,17 +14,17 @@ export default class FShowCoupon extends Feature {
         const couponDate = coupon.valid && coupon.valid.replace(/\[date](.+)\[\/date]/, (m0, m1) => { return new Date(m1 * 1000).toLocaleString(); });
 
         HTML.beforeBegin("#game_area_purchase",
-            `<div class="early_access_header">
+            `<div class="early_access_header es_coupon_info">
                 <div class="heading">
                     <h1 class="inset">${Localization.str.coupon_available}</h1>
                     <h2 class="inset">${Localization.str.coupon_application_note}</h2>
                     <p>${Localization.str.coupon_learn_more}</p>
                 </div>
                 <div class="devnotes">
                     <div style="display:flex;padding-top:10px">
-                        <img src="http://cdn.steamcommunity.com/economy/image/${coupon.image_url}" style="width:96px;height:64px;"/>
+                        <img src="//steamcommunity-a.akamaihd.net/economy/image/${coupon.image_url}" style="width:96px;height:64px;">
```

Comment:
I'm thinking. Perhaps we should add steam's cdn option to the global config. So we can easily change it, or even dynamically set it if one cdn is down.